enzy_htp/_config/xtb_config.py

```diff
@@ -51,6 +59,9 @@ class XTBConfig(BaseConfig):
     SUPPORTED_XTB_THEORY_LEVELS:List[str] = "GFN0 GFN1 GFN2".split()
     """Allowed levels of theory for xtb to use."""
 
+    FORCE_CONSTANT:Union[None,float]=None
```

Comment:
let be DEFAULT_FORCE_CONSTANT. Avoid Union[None,...] and use Optional[...] instead

Reply:
I don't see the point of having a None value in the Config class